src/components/ActionsResolvedCard.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {
+    Skeleton
+} from '@red-hat-insights/insights-frontend-components';
+
+import {
+    Card, CardHeader, CardBody,
+    Progress, ProgressMeasureLocation,
+    Level, LevelItem
+} from '@patternfly/react-core';
+
+function buildBody (status) {
+    if (status.status !== 'fulfilled') {
+        return <Skeleton size='md' />;
+    }
+
+    const { resolved, total } = status.data.summary;
+
+    return <Progress
+        value={ resolved }
+        max={ total }
+        label={ `${resolved} of ${total}` }
+        measureLocation={ ProgressMeasureLocation.outside } />;
+}
+
+const ActionsResolvedCard = ({ status }) => (
+    <Card className='ins-c-card__actions-resolved'>
+        <CardHeader>
+            <Level>
```

Comment:
Does this need to be a level? if we're following PF, we should just use "Actions Resolved" as the `title` prop.

Reply:
@ryelo I am not following. I just split out this https://github.com/RedHatInsights/insights-remediations-frontend/blob/master/src/routes/RemediationDetails.js#L96 into a standalone component.

Reply:
brain 💩